scrapers/in/events.py

```diff
@@ -1,90 +1,97 @@
-import pytz
-import cloudscraper
-import dateutil.parser
+import logging
+import os
 from datetime import date
-import lxml.html
+
+import dateutil.parser
+import pytz
+import requests
 from openstates.scrape import Scraper, Event
+from .utils import add_space
+
+log = logging.getLogger(__name__)
 
 
 class INEventScraper(Scraper):
     _tz = pytz.timezone("America/Indianapolis")
     # avoid cloudflare blocks for no UA
     cf_headers = {
```

Comment:
I don't think we need these headers now that we aren't using the direct site any more.

Reply:
We need it, without it we get a 403 forbidden url error